src/collection-view.js

```diff
@@ -207,11 +207,13 @@ const CollectionView = Backbone.View.extend({
   // Render children views. Override this method to provide your own implementation of a
   // render function for the collection view.
   render() {
-    if (this._isDestroyed) { return this; }
-    this.triggerMethod('before:render', this);
-    this._renderChildren();
-    this._isRendered = true;
-    this.triggerMethod('render', this);
+    if (!this._isDestroyed) {
```

Comment:
This part came up when I ran the code coverage test. It listed collectionview at 99.98%.
I can look into it again in a separate issue. Just thought I'd bring it up.